contracts/seed/Seed.sol

```diff
@@ -322,26 +313,45 @@ contract Seed {
                 _classCaps.length == _classFee.length,
             "Seed: All provided arrays should be same size");
         for(uint8 i = 0; i < _classCaps.length; i++){
-            require(
-                endTime < _classVestingStartTime[i],
-                "Seed: vesting start time can't be less than endTime"
-            );
-            require(
-                _classFee[i] < MAX_FEE,
-                "Seed: fee cannot be more than 45%"
-            );
-
-            classes.push(ContributorClass(
+            checkAndPush(
```

Comment:
There is a simpler way, we can just call `addClass` from here, which already does the thing, instead of using another function => increasing gas usage for deployment